.github/workflows/tm.integration.tests.yml

```diff
@@ -52,3 +52,10 @@ jobs:
         uses: ./.github/actions/tm-integration-tests
         env:
           GOPATH: /github/workspace
+      - name: go-version
```

Comment:
This can be a human-readable name instead of *go-version*.

Reply:
Changed to "Check Go Version"